app/components/Doc.tsx

```diff
@@ -21,7 +21,7 @@ export function Doc({
       <div className="h-4" />
       <div className="h-px bg-gray-500 opacity-20" />
       <div className="h-4" />
-      <div className="prose prose-gray prose-sm dark:prose-invert max-w-none">
+      <div className="prose prose-gray prose-sm dark:prose-invert max-w-none md:grid">
```

Comment:
Using a breakpoint here, since the existing styling for mobile needs it to not bork things.